chat-widget/src/common/telemetry/TelemetryConstants.ts

```diff
@@ -141,6 +144,11 @@ export enum TelemetryEvent {
     CustomerVoiceResponsePageLoaded = "CustomerVoiceResponsePageLoaded",
     CustomerVoiceFormResponseSubmitted = "CustomerVoiceFormResponseSubmitted",
     CustomerVoiceFormResponseError = "CustomerVoiceFormResponseError",
+    BotAuthActivityEmptySasUrl = "BotAuthActivityEmptySasUrl",
```

Comment:
Please update Telemetry.md with all newly added TelemetryEvent and BroadcastEvent (the one we raise only. Don’t need to log the one we subscribe to, I.e. ExecuteSigninCardCallbackResponse)

Reply:
done